addons-l10n/en/op-badge.json

```diff
@@ -0,0 +1,3 @@
+{
+  "op-badge/op": "Original poster"
```

Comment:
Should this be changed to `Project author` (or similar) in project pages?

Reply:
i think so too